shadow-cljs.edn

```diff
@@ -1,6 +1,10 @@
 {:deps {:aliases [:cljs]}
  :builds
  {:node
-  {:target      :node-library,
-   :exports-var nuid.zk.js/exports,
-   :output-to   "target/node/nuid_zk.js"}}}
+  {:target      :node-library
+   :exports-var nuid.zk.js/exports
+   :output-to   "target/node/nuid_zk.js"}
```

Comment:
if we're going to use branches to differentiate the RN platform (which i think is looking pretty good), i think we can just have a single build for each branch

Reply:
Good point, will remove.